fossil-add: add page (#7108)
tldr-pages/tldr: pages/linux/fossil-add.md

# fossil add

> > Put files or directories under Fossil version control.

Author: > I'm not sure if that's correct. If you change a file which already is under version control, you still have to add it, right?

No, that's not the case in Fossil.  ie, Fossil does not have the Git concept of the index/staging-area.  When you commit in Fossil, all pending changes are committed.  Thus, add and remove are solely concerned with what is or is not tracked under version control (though with remove you can also optionally also actually remove the file from the local checkout).

That said, if my verbiage is confusing to you, it will likely be so to others.  Given my comments above, would you still like to me to proceed with your suggested change?  I have no objections...  Thoughts @marchersimon ?

> > Put files or directories under Fossil version control.

Reviewer: In this case your wording is perfectly fine. I just thought it was more similar to git.

> More information: <https://fossil-scm.org/home/help/add>.

> - Stage a file or directory for the next commit to the current checkout:

Reviewer: This makes it sound like it works like git. How about this?
```suggestion
- Put a file or directory under version control, so it will be in the current checkout:
```
Or similar. But `stage a file` sounds like it is already under version control.

Author: Excellent point, thanks!

`fossil add {{path/to/file_or_directory}}`

> - Undo all add commands issued since the last commit to the current checkout:

Reviewer: Sounds a bit too complicated
```suggestion
- Remove all added files from the current checkout:
```
